src/import/load-costume.js

```diff
@@ -176,7 +176,7 @@ const fetchBitmapCanvas_ = function (costume, runtime, rotationCenter) {
                 assetMatchesBase: scale === 1 && !textImageElement
             };
         })
-        .catch(() => {
+        .finally(() => {
```

Comment:
Errors will get caught below, but this change does mean that we will no longer be swallowing errors with loading the text layer... the costume will turn into a gray question mark if the text layer fails to load, but separating out the intermingled code paths was seeming a little more risky than displaying a gray question mark if the text layer failed to load...

Reply:
Hmm... looks like `finally` isn't being accepted by `tap`. Maybe it's only supported by newer Node? I'm having trouble looking up info on that.